addons/im_livechat/models/discuss_channel_member.py

```diff
@@ -71,16 +72,40 @@ def _compute_agent_expertise_ids(self):
     def _create_or_update_history(self, values_by_member=None):
         if not values_by_member:
```

Comment:
Not directly related: we might want to make `values_by_member` required as we always give some now. This way we avoid the existence check.